src/pages/island-architecture.md

```diff
@@ -1,5 +1,240 @@
 ---
```

Comment:
@FredKSchott since we removed the single page for the island architecture, where do you think this content would fit better?

Reply:
Maybe this would fit as an explanation/discussion page?

Reply:
@aFuzzyBear I will review the rest of this file after we identify where it should go to.

Reply:
Awesome, I left the finale to be written up with more of stronger tie into Astro, tbh Im gonna go edit part of this to see if I can remove some of the superfluous commentary in it, a home for this would be good, there isnt very much literature out there about Island architecture atm.